css/wishlist.css

```diff
@@ -0,0 +1,82 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap");
+
+* {
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+  padding: 0;
+  color: var(--secondary-dark);
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+}
+
+.wishlist-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+.wishlist-card {
+  height: 23rem;
+  width: 18rem;
+  border-radius: 5px;
+  margin: 1rem;
+  border: 1px solid var(--light-grey-border);
+}
+.container-img {
+  position: relative;
+  width: 100%;
+  height: 65%;
+  overflow: hidden;
+}
+.imgg {
+  position: absolute;
+  height: 100%;
+  width: 100%;
+  object-fit: cover;
+}
+.rating i {
+  color: #ecc722;
```

Comment:
Suggestion: Can store it as a variable in wishlist.css as icon-color and use it!!